src/components/rightCol/ThisWeek.vue

```diff
@@ -39,7 +40,10 @@
       },
       isLoadingTransactions(): boolean {
         return this.$store.state.lokapi.transactionsLoading
-      }
+      },
+      hasTransactionsLoadingError(): boolean {
+        return this.$store.state.lokapi.transactionsLoadingError
+      },
```

Comment:
Same remark

Reply:
Yes, same answer as above 😄

Reply:
Same here. If you have a suggestion on how to do it, please advise.